config/tools/tools.go

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2023. Nydus Developers. All rights reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package tools
+
+import "os"
+
+func IsDirExisted(path string) (bool, error) {
```

Comment:
Can we put such helper functions into files named as utils.go ?

Reply:
Done.